tensorboard/plugins/histogram/metadata.py

```diff
@@ -40,4 +41,13 @@ def create_summary_metadata(display_name, description):
 
 
 def parse_summary_metadata(content):
+  """Parse summary metadata to a Python object.
+
+  Arguments:
+    content: the `content` field of a `SummaryMetadata` proto
```

Comment:
Thank you! I try to follow this convention but sometimes forget. I will fix this.